buildsystem/test_dependencies.gradle

```diff
@@ -0,0 +1,18 @@
+dependencies {
+    // We specify junit before anything else to make sure that this version is prioritized over the
+    // one bundled in the android.jar
+    testImplementation "junit:junit:${versions.junit}"
+    testImplementation "io.mockk:mockk:${versions.mockk}"
+    testImplementation 'com.jraska.livedata:testing-ktx:1.1.0'
+    testImplementation "org.jetbrains.kotlinx:kotlinx-coroutines-test:${versions.kotlinx_coroutines}"
+    testImplementation "androidx.arch.core:core-testing:2.1.0"
+
+    testImplementation "org.jetbrains.kotlin:kotlin-reflect:${versions.kotlin}"
+    testImplementation "com.github.gnosis.svalinn-kotlin:utils-testing:${versions.svalinn}"
+    testImplementation "org.mockito:mockito-core:${versions.mockito}"
+    testImplementation "org.mockito:mockito-inline:${versions.mockito}"
+    testImplementation "org.powermock:powermock-module-junit4:${versions.powermock}"
+    testImplementation "org.powermock:powermock-api-mockito2:${versions.powermock}"
+    testImplementation "org.powermock:powermock-core:${versions.powermock}"
+    testImplementation "org.powermock:powermock-module-junit4-rule:${versions.powermock}"
```

Comment:
I doubt we need powermock and mockito having mockk. We can do statics with it too.

Reply:
removed them for now